modules/rbac-viewing-local-roles.adoc

```diff
@@ -14,7 +14,7 @@ You can use the `oc` CLI to view local roles and bindings by using the
 * Obtain permission to view the local roles and bindings:
 
 ifdef::openshift-dedicated[]
-** Users with the *dedicated-cluster-admin* role can view and manage local roles and bindings. 
+** Users with the *dedicated-cluster-admin* role can view and manage local roles and bindings.
```

Comment:
\`dedicated-cluster-admin\` - we changed the formatting guidelines to almost always prefer just \` \`.

Reply:
Thanks for the notice! I changed it here and in the other sections for this topic.